pkg/loop/reportingplugins/loopp_service_test.go

```diff
@@ -10,6 +10,9 @@ import (
 	"github.com/smartcontractkit/chainlink-common/pkg/loop"
 	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal"
 	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/test"
+	reportingplugin_test "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/test/ocr2/reporting_plugin"
+	resources_test "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/test/resources"
+	coreapi_test "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/test/resources/core_api"
```

Comment:
I'm not a fan of underscores in package names, but let's align the aliases either way:
```suggestion
	core_api_test "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/test/resources/core_api"
```
```suggestion
	coreapi_test "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/test/resources/coreapi"
```

Reply:
https://go.dev/blog/package-names

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps.